### PR TITLE
add release action job

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: CHANGELOG/CHANGELOG-v0.1.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Changelog 
+All notable changes to this project will be documented in this file.
+
+## v0.1.0
+
+### New Features
+* Enable placement controller.
```
Comment (elgnay) [inline CHANGELOG/CHANGELOG-v0.1.md:7]: How about "Selecting managed cluster with cluster labels/claims."

Comment (zhiweiyin318) [reply CHANGELOG/CHANGELOG-v0.1.md]: done

**Pull request merged**